lib/src/errors/dson_exception.dart

```diff
@@ -9,13 +13,13 @@ class DSONException implements Exception {
   /// Exception from DSON
   DSONException(this.message, [this.stackTrace]);
 
-  String get _className => 'DSONException';
+  String get _className => '[DSONException]';
```

Comment:
É interessante usar uma interpolação com o tipo da classe. Dessa forma, se o nome da classe for alterado essa string será considerada como referência da classe e também será alterada.
```suggestion
  String get _className => '[$DSONException]';
```

Reply:
mano faz todo sentido!! ótima idéia

Reply:
done e03bbfcf12b3910cf2a821e421f775da30144206